projects/Mallard/fastlane/Fastfile

```diff
@@ -53,14 +53,17 @@ platform :ios do
       readonly: true,
       git_url: "https://github.com/guardian/code-signing.git",
       keychain_name: "CI",
-      keychain_password: ENV["FASTLANE_PASSWORD"]
+      keychain_password: ENV["FASTLANE_PASSWORD"],
+      skip_provisioning_profiles: true
     )
 
     update_code_signing_settings(
       use_automatic_signing: false,
       path: "ios/Mallard.xcodeproj",
     )
 
+    get_provisioning_profile(app_identifier: "uk.co.guardian.gce", readonly: true, api_key: appstore_api_key)
```

Comment:
Maybe it would be good to have a comment here, why we are using ios live app's package name so in the future it does not confuse devs

Reply:
@mohammad-haque this is the bundle identifier for the editions app

Reply:
ah, of course. thanks, @aoifemcl15